src/ActionModal/components/ActionModal.vue

```diff
@@ -731,14 +730,14 @@ export default {
           overview(networkId: $networkId, address: $address) {
             totalRewards
             liquidStake
-            totalStake
             accountInformation {
               accountNumber
               sequence
             }
           }
         }
       `,
+      fetchPolicy: "cache-and-network",
```

Comment:
very cool. can we set this globablly or does it have to be set per component?

Reply:
we can, but may want to not do it for things like `networks`. but we could make it the default and set the fetch policy for networks to `cache`